scipy/optimize/minpack.py

```diff
@@ -62,7 +62,7 @@ def fsolve(func, x0, args=(), fprime=None, full_output=0,
         The starting estimate for the roots of ``func(x) = 0``.
     args : tuple, optional
         Any extra arguments to `func`.
-    fprime : callable(x), optional
+    fprime : callable(x, *args), optional
```

Comment:
This breaks documentation build. You can change to
```
callable ``f(x, *args)``
```
as it is for `func` argument above.